src/Command/Command.php

```diff
@@ -283,7 +283,7 @@ protected function getTableHelper()
         return $table
             ->setRows([])
             ->setLayout(TableHelper::LAYOUT_BORDERLESS)
-            ->setHorizontalBorderChar('')
+            ->setHorizontalBorderChars('')
```

Comment:
`getTableHelper` is only called for ~2.4 TableHelpers, which uses `setHorizontalBorderChar`:

https://github.com/symfony/console/blob/2.4/Helper/TableHelper.php#L224

Reply:
Should we maybe then "deprecate" this helper call ? 2.4 (even 2.x) is not maintained anymore...

Reply:
(Left it as is then)